src/bench/bench.cpp

```diff
@@ -61,6 +59,11 @@ void benchmark::BenchRunner::RunAll(const Args& args)
 
         Bench bench;
         bench.name(p.first);
+        if (args.min_duration > std::chrono::milliseconds::zero()) {
+            // convert to nanos before dividing to reduce rounding errors
+            bench.minEpochTime(std::chrono::nanoseconds(args.min_duration) / bench.epochs());
```

Comment:
Looking at https://en.cppreference.com/w/cpp/chrono/duration/duration_cast, would something like this be a good idea?

```cpp
// converting integral duration to integral duration of shorter divisible time unit:
    // no duration_cast needed
    std::chrono::duration<long, std::micro> int_usec = int_ms;
```

also line 62, can shorten if preferred
```diff
-        if (args.min_duration > std::chrono::milliseconds::zero()) {
+        if (args.min_duration > 0ms) {
```

also, where `std::chrono` is added, look at include headers for `<chrono>` where needed

Reply:
hm, I think it's fine to use `std::chrono::nanoseconds`, it's basically the same but with `int64_t`. It should be large so there's no easy chance for overflow. I generally try to not use `duration_cast`, because that's a sign that at that place there will be some loss of information due to the casting.

I've changed  line 64 to 0ms in b8f16c323e02c3d06e5ad57b6c0f96196d9b12e0